packages/storybook/stories/NonReact/NonReactComposites.stories.mdx

```diff
@@ -43,7 +43,7 @@ const callAdapter = await callComposite.loadCallComposite(
   //The for composite which allows you control composite externally,
   // This function will be exposed to window.callComposite once you import composite bundle
   {
-    groupId, // string, Provide any GUID to join a group
+    locator: { groupId: groupId }, // string, Provide any GUID to join a group
```

Comment:
Maybe we can add a link to explain what locator is? It could be groupId or TeamsLink or something else, but not necessary needed in this PR

Reply:
I should definitely update the comment. Good point!